src/communication/v2/session.hpp

```diff
@@ -213,6 +225,10 @@ class WebsocketSession : public std::enable_shared_from_this<WebsocketSession<TS
     if (!IsConnected()) {
       return;
     }
+
```

Comment:
I am not sure why do you `DecrementCounter` for `ActiveSessions` when you don't `IncrementCounter` for `ActiveSessions` in `DoAccept`

Reply:
This is a `WebsocketSession`. I increment the counter in `Session::Start() (line 279)`, and there we switch to `WebsocketSession` in `Session::OnRead`